admin/package-files/docker-plugin/upstart/flocker-docker-plugin.conf

```diff
@@ -0,0 +1,11 @@
+# flocker-docker-plugin - flocker-docker-plugin job file
+
+description "Flocker Docker Plugin"
+author "ClusterHQ <support@clusterhq.com>"
+
+start on starting docker
+stop on stopped docker
+respawn
+
+# Start the process
+exec /usr/sbin/flocker-docker-plugin --logfile=/var/log/flocker/flocker-docker-plugin.log
```

Comment:
I notice a lock / dangling symlink along side the socket which points to a file named after the process ID of the flocker-docker-plugin:

```
root@acceptance-test-richardw-0:~# ls -l /run/docker/plugins/flocker/
total 0
srw------- 1 root root 0 Aug 18 09:36 flocker.sock
lrwxrwxrwx 1 root root 4 Aug 18 09:36 flocker.sock.lock -> 6229
``` 

Perhaps that's just how Linux tracks the owner of the socket?